src/App.js

```diff
@@ -25,20 +27,61 @@ const StyledContent = styled(Content)`
 `;
 
 function App() {
+  const [rawList, setRawList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const fetchCvList = async () => {
+    setLoading(true);
+    try {
+      const response = await fetch("/cvlist");
+      const obj = await response.json();
+      console.log(response);
+      console.log(obj);
+      if (obj.confirmation === "fail") {
+        throw new Error();
+      }
+      obj.data.forEach(cv => (cv.key = cv._id));
+      setRawList(obj.data);
+      setLoading(false);
+    } catch (e) {
+      setLoading(false);
+      openNotification({
+        type: "error",
+        message: "Не удалось загрузить данные"
+      });
+    }
+  };
+  const [favorites, setFavorites] = useState([]);
   return (
     <Router>
       <Layout>
-        <StyledHeader
-          theme="light"
-        >
+        <StyledHeader theme="light">
           <TopMenu />
         </StyledHeader>
         <StyledDiv>
           <StyledContent>
             <Switch>
               <Route exact path="/" component={Home} />
-              <Route path="/favorites" />
-              <Route path="/list" component={CvTable} />
+              <Route
+                path="/favorites"
+                component={() => (
```

Comment:
check why do you need ()=>

Reply:
i need it for passing props from parent component
i have changed Favorite, but in CvList props are nesesery